src/NBitcoin/BitcoinStream.cs

```diff
@@ -81,19 +81,19 @@ public Stream Inner
             }
         }
 
-        private readonly bool _Serializing;
-        public bool Serializing
-        {
-            get
-            {
-                return _Serializing;
-            }
-        }
+		private readonly bool _Serializing;
```

Comment:
Whilst we are in this area, you might as well rename to `serializing`

Reply:
Done